plugins/table/src/ReactDataTable.tsx

```diff
@@ -236,15 +237,16 @@ export default function ReactDataTable(props: DataTableProps) {
               const keyIsMetric = metricsSet.has(key);
               const text = cellText(key, format, val);
               const isHtml = !keyIsMetric && isProbablyHTML(text);
+              const showCellBar = keyIsMetric && !hideCellBars;
               return (
                 <td
                   key={key}
                   // only set innerHTML for actual html content, this saves time
                   dangerouslySetInnerHTML={isHtml ? { __html: text } : undefined}
                   data-sort={val}
-                  className={keyIsMetric ? 'text-right' : ''}
+                  className={showCellBar ? 'text-right' : ''}
```

Comment:
It probably makes sense to always align metrics to the right even if we don't need to show the bar chart?

Reply:
Sure, that's fine... it looks more wrong than right to me to have them right-aligned in many cases, but that all depends on the number formatting. I guess there's no need to squeeze that extra change, as it wasn't part of the original issue. I'll change it back.